Content.Server/Speech/EntitySystems/SkeletonAccentSystem.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Text.RegularExpressions;
+using Content.Server.Speech.Components;
+using Robust.Shared.Random;
+
+namespace Content.Server.Speech.EntitySystems;
+
+public sealed class SkeletonAccentSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    private static readonly Dictionary<string, string> DirectReplacements = new()
+    {
+        { "fuck you", "I've got a BONE to pick with you" },
+        { "fuck", "rattle rattle" },
+        { "shit", "rattle rattle" },
+        { "definitely", "make no bones about it" },
+        { "absolutely", "make no bones about it" },
+        { "lonely", "bonely"}
+    };
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<SkeletonAccentComponent, AccentGetEvent>(OnAccentGet);
+    }
+
+    public string Accentuate(string message, SkeletonAccentComponent component)
+    {
+        // Order:
+        // Do replacements first
+        // Then prefix/suffix
+
+        var msg = message;
+
+        foreach (var (first, replace) in DirectReplacements)
+        {
+            msg = Regex.Replace(msg, $@"(?<!\w){first}(?!\w)", replace, RegexOptions.IgnoreCase);
+        }
+
+        // Suffix
+        if (_random.Prob(component.ackChance))
+        {
+            msg += " ACK ACK!";
```

Comment:
localize. ik not all accents do this but might as well here for a static string

Reply:
~~¿Ack Ack?~~ Yep, can do. Should I make a generic .ftl, that future AccentSystems can share?

Reply:
I did so on my latest commit and made it generic. Thanks! Marking resolved.